managed/ui/src/components/schedules/Schedules.js

```diff
@@ -75,6 +75,19 @@ class ScheduleDisplayItem extends Component {
       !schedule.taskParams.timeBeforeDelete || schedule.taskParams.timeBeforeDelete === 0
         ? 'Unlimited'
         : moment.duration(schedule.taskParams.timeBeforeDelete).humanize();
+    
+    let backupType = '';
```

Comment:
Can we declare all the variables on the top?

Reply:
Done